AlaqmaMaryamZadibaKokweh/MALZ032.xml

```diff
@@ -0,0 +1,144 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="MALZ032" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title> Vita of Zamikāʾel ʾAragāwi and other texts dedicated to him</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DN"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <pubPlace>Hamburg</pubPlace>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine
+                    multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS1012AddiLaqmaMaryam"/>
+                        <idno>MALZ-032</idno>
+                    </msIdentifier>
+                    <msContents>
+                        <summary/>
+
+                        <msItem xml:id="ms_i1">
+                            <title type="complete" ref="LIT1526GadlaZ">Vita of Zamikāʾel ʾAragāwi</title>
+                            <textLang mainLang="gez"/>
+                            
+                        </msItem>                        
+                    </msContents>
```

Comment:
and what are the other texts?

Reply:
I see only the end of Malke' at the end, but there must be smth inbetween.

Reply:
it was just because the title says "Vita of Zamikāʾel ʾAragāwi and other texts dedicated to him" 
maybe then leave just VZA, it is the main text anyway